src/Microsoft.SqlTools.ServiceLayer/TableDesigner/TableDesignerService.cs

```diff
@@ -1845,14 +1845,15 @@ private Dac.TableDesigner GetTableDesigner(TableInfo tableInfo)
         private void UpdateTableTitleInfo(TableInfo tableInfo)
         {
             var td = GetTableDesigner(tableInfo);
-            var advancedOpsIndex = tableInfo.Tooltip.LastIndexOf('[');
-            var advancedOps = "";
-            if(advancedOpsIndex > -1){
-                advancedOps = " " + tableInfo.Tooltip.Substring(advancedOpsIndex);
-            }
+            // Disabled due to issues with title generation.
```

Comment:
Just delete this stuff IMO - as Cheena pointed out in the other one we can always revert a commit or refer back to the original code if/when we add this back in.